Makefile

```diff
@@ -0,0 +1,12 @@
+DOCKER?=docker
+
+.PHONY: azure-image
+azure-image: reproducible-build/artifacts/lighthouse tdx-poky
+	mkdir -p build && chmod 0777 ./build
+	mkdir -p reproducible-build/artifacts && chmod 0777 reproducible-build/artifacts
+	$(DOCKER) run --rm -it -v $(CURDIR)/reproducible-build/artifacts:/artifacts -v $(CURDIR)/build:/build tdx-poky
+	chmod 0755 build reproducible-build/artifacts
```

Comment:
why are we setting the permissions here again?
In line 6, they are being set 777 and here changed back to 755. Maybe do it from the start 755